app/src/main/res/color/component_color_shared_survey_option_selector.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:color="@color/component_color_survey_radio_button_unchecked_color" android:state_checked="false" android:state_enabled="false" />
+  <item android:color="@color/component_color_survey_radio_button_unchecked_color" android:state_checked="false" android:state_enabled="true" />
+  <item android:color="@color/component_color_survey_radio_button_checked_color" android:state_checked="true" />
+</selector>
```

Comment:
Missing newline at end?

Reply:
I've added newlines to the end of all resource files.